.github/workflows/main.yml

```diff
@@ -38,7 +38,7 @@ jobs:
      - name: Run presubmit tests
        run: |
             go env -w GOFLAGS=-mod=mod
-            make -j 5 all
+            make -j 12 all
```

Comment:
have you tried using "make -j`nproc`" to dynamically retrieve the number of processors?

Reply:
Thanks for the suggestion. `nproc` worked great!